lib/commands/plugin/fetch-plugin.ts

```diff
@@ -1,46 +1,14 @@
 ///<reference path="../../.d.ts"/>
 "use strict";
 
-import service = require("../../services/cordova-plugins");
-import validUrl = require("valid-url");
-
 export class FetchPluginCommand implements ICommand {
-	constructor(private $logger: ILogger,
-				private $fs: IFileSystem,
-				private $cordovaPluginsService: service.CordovaPluginsService,
-				private $stringParameterBuilder: IStringParameterBuilder) {
-	}
+	constructor(private $pluginsService: IPluginsService,
+				private $stringParameterBuilder: IStringParameterBuilder) {}
 
-	public allowedParameters = [this.$stringParameterBuilder.createMandatoryParameter("You must specify local path, URL to a plugin repository, name or keywords of a plugin published to the Cordova Plugin Registry.")];
+	public allowedParameters = [this.$stringParameterBuilder.createMandatoryParameter("You must specify local path, URL to a plugin repository, name or keywords of a plugin published to the NPM.")];
```

Comment:
We've talked about this offline but I'm leaving a comment so that it won't be forgotten.
This parameter can be made non-mandatory and each of the plugin-services can handle validation on their own. That way each service can have a unique message.